webapp/src/components/ContractActionForm/index.js

```diff
@@ -261,7 +261,7 @@ const ContractActionForm = ({ accountName, action, abi, onSubmitAction }) => {
           color="primary"
           onClick={handleSubmit}
         >
-          {t('executeTransaction')}
+          {t('Execute Transaction')}
```

Comment:
I think you need to check the language files, to see if that entry exists.
Did you check that the text changes when you switch to Spanish?

Reply:
good catch